generated-docs/Graphics/Canvas.md

```diff
@@ -580,23 +580,23 @@ Get the current font.
 #### `setFont`
 
 ``` purescript
-setFont :: forall eff. String -> Context2D -> Eff (canvas :: CANVAS | eff) Context2D
+setFont :: forall eff. String -> Context2D -> Eff (canvas :: CANVAS | eff) Unit
 ```
 
 Set the current font.
 
 #### `fillText`
 
 ``` purescript
-fillText :: forall eff. Context2D -> String -> Number -> Number -> Eff (canvas :: CANVAS | eff) Context2D
+fillText :: forall eff. Context2D -> String -> Number -> Number -> Eff (canvas :: CANVAS | eff) Unit
```

Comment:
Did you miss this one?

Reply:
I mean, is there a reason why `Context2D` isn't last here like it is in the other methods?

Reply:
yep

Reply:
wait that's in the generated docs. it is correct in the code. i'll take a look

Reply:
context2d should be first. that's what we discussed before. it's just that the docs aren't updated for the others. i'll commit a document update

Reply:
that's done. i hadn't realized generating documents is not part of the build